src/Scaffold_Command.php

```diff
@@ -886,6 +1002,16 @@ private function get_textdomain( $textdomain, $args ) {
 		return 'YOUR-TEXTDOMAIN';
 	}
 
+	/**
+	 * Generate the machine name for function declarations.
+	 *
+	 * @param string $slug Slug name to .
+	 * @return string
+	 */
+	private function generate_machine_name( $slug ) {
+		return preg_replace( '/-/', '_', $slug );
```

Comment:
Is there a reason not to use `str_replace()`, which is more performant?

Reply:
I used refactor method feature in my IDE and didn't look at the code. You are totally right, I will update.